examples/api_intro.ipynb

```diff
@@ -5093,7 +5093,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.12"
+   "version": "3.10.0"
```

Comment:
3.8.12 been used for all the notebooks and stated as a recommendation; there were problems with some dependencies in higher Python versions.